chips/cc26x0/src/uart.rs

```diff
@@ -462,11 +462,7 @@ impl peripheral_manager::PowerClient for UART {
         self.configure();
     }
 
-    fn lowest_sleep_mode(&self) -> u32 {
-        if self.rx_remaining_bytes.get() == 0 {
-            chip::SleepMode::DeepSleep as u32
-        } else {
-            chip::SleepMode::Sleep as u32
-        }
+    fn lowest_sleep_mode(&self) -> u32 {      
+        chip::SleepMode::Sleep as u32
```

Comment:
Why did you change this back? This way, it'll never be able to enter deep sleep since UART is registered as a peripheral